### PR TITLE
Section content

### PR DESCRIPTION
**Pull request by paigewilliams**
Here is a first pass at adding content. I am thinking that we can pass a different classname to the `Section` component to determine the layout of the container.

### CODE REVIEW

File: client/src/components/cascadeSite/Slides/Section.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import { createSectionRefAction } from "../../../actions/slides";
+
+import "./Slides.scss";
+
+const Section = ({ slide, children, dispatch }) => {
+  const sectionRef = useRef(null);
+  useEffect(() => {
```
Comment (timhitchins) [inline client/src/components/cascadeSite/Slides/Section.js:9]: Love the simplicity of this!

File: client/src/components/cascadeSite/Slides/Slides.scss [hunk 2]
```diff
@@ -92,10 +93,33 @@
...
+    background-color: rgba(255, 255, 255, 0.7);
+  }
+  h1 {
+    font-size: 3em;
+    color: black;
+    max-width: 70rem;
+    padding: 1rem;
+    width: fit-content;
+    text-transform: uppercase;
+    background-color: rgba(255, 255, 255, 0.7);
```
Comment (timhitchins) [inline client/src/components/cascadeSite/Slides/Slides.scss]: Might be worth trying slightly less opacity, maybe .5?  I am open though.

File: client/src/components/cascadeSite/Slides/Slides.scss [hunk 3]
```diff
@@ -92,10 +93,33 @@
...
+  }
+  p {
+    font-size: 1.4em;
```
Comment (timhitchins) [inline client/src/components/cascadeSite/Slides/Slides.scss]: Will probably need to adjust font-size for mobile.

File: client/src/config/imgConfig.js [hunk 4]
```diff
@@ -3,41 +3,77 @@ export const imageConfig = [
     imageURI:
       "https://nca-toolkit.s3-us-west-2.amazonaws.com/OregonLive_hazepdxjpg-e8d26d8245f61043.jpg",
     altText: "person running",
+    heading: "Oregon has toxic air pollution",
+    body:
+      "<strong>We all have the right to breathe clean air.</strong> However, in the Portland metro area, the presence of fine and ultra-fine particulate matter from older dirty diesel engines makes our air unhealthy. <strong>Clackamas, Multnomah, and Washington counties rank in the top 5 percent of all counties nationwide for ambient diesel particulate concentrations</strong> and have the highest exposure rate of all counties in Oregon.",
```
Comment (timhitchins) [inline client/src/config/imgConfig.js]: Love what you did with this (adding the content to the config object).

**Review by timhitchins — COMMENTED** [review]
@paigewilliams Thank you!  Just leaving general feedback.  It is so awesome to see it coming together.  I added some feedback on specific lines.  I also like your idea about passing a different `className` to the `Section` component. 
On a different note, how do you want to handle mobile vs desktop media queries?  This is looking good on over 600px but getting smashed around 400px width.  Maybe change font size to percentages?  I am open to ideas.  
image

Hope that helps.  Lemme know if you need some more feedback.  THANK YOU!

File: client/src/components/cascadeSite/Slides/Slides.scss (hunk 3 above)
Comment (paigewilliams) [reply client/src/components/cascadeSite/Slides/Slides.scss]: I can add some font sizes for mobile and desktop.

**Comment by timhitchins** [comment]
@paigewilliams These updates look great!  I would say merge this and I will merge master into my #6.   I really love what you did with the headings too! 
image

**Pull request merged**